src/App.js

```diff
@@ -1,13 +1,16 @@
 import React, { Component } from 'react';
 import Users from './components/Users/users';
+import User from './components/User/user';
+import Post from './components/Post/post';
 
 class App extends Component {
+
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <Users/>
-        </header>
+        <Users/>
```

Comment:
Maybe we could use something more descriptive like `UserList`

Reply:
Got it